pkg/cri/resource-manager/policy/builtin/static-pools/stp-policy.go

```diff
@@ -136,7 +110,10 @@ func (stp *stp) Start(add []cache.Container, del []cache.Container) error {
 	var err error
 
 	if stp.conf == nil {
-		return stpError("cannot start without any configuration")
+		err := stp.setConfig(cfg)
+		if err != nil {
+			return err
```

Comment:
Out of curiosity, is there a particular reason you 1) don't use the pre-declared `err` in this branch of code (I can't see any apparent reason based on the control flow), and 2) you don't use a combined assign-and-test here while for an almost identical piece of code you do use that later in `configNotify()`. IOW, why not also have this here:
```
    if err = stp.setConfig(cfg); err != nil {
        return err
    }
```

Reply:
Well spotted, thanks. Bad style, caused by blindly copying code around. Now fixed. Also changed the whole function to use combined assign-and-test in unified way